src/main/java/com/gmail/goosius/siegewar/command/SiegeWarCommand.java

```diff
@@ -91,25 +108,35 @@ private void parseSiegeWarGuideCommand(Player player) {
 		
 	}
 
-	private void parseSiegeWarNationCommand(Player player, String[] args) {
-		if (!player.hasPermission(SiegeWarPermissionNodes.SIEGEWAR_COMMAND_SIEGEWAR_NATION.getNode(args[0]))) {
-			player.sendMessage(Translation.of("msg_err_command_disable"));
-			return;
+	private void parseSiegeWarCollectCommand(Player player) {
+		int incomeTypesCollected = 0;

```

Comment:
There should be a test for TownyEconomyHandler.isActive() before any sort of money code is used.

Reply:
Fixed